package-lock.json

```
@@ -58,9 +58,9 @@
 			}
 		},
 		"node_modules/@preact/signals-core": {
-			"version": "1.2.1",
-			"resolved": "https://registry.npmjs.org/@preact/signals-core/-/signals-core-1.2.1.tgz",
-			"integrity": "sha512-aqzRMNFU1hoQyP4Kb1ldJrUTCnA9vqPDa7qHEQzHJ3upnBOcC2pjmvjAuTqGuY4AVTtUkCQV0FvOCuIQQ2hSdA==",
+			"version": "1.3.1",
+			"resolved": "https://registry.npmjs.org/@preact/signals-core/-/signals-core-1.3.1.tgz",
+			"integrity": "sha512-DL+3kDssZ3UOMz9HufwSYE/gK0+TnT1jzegfF5rstgyPrnyfjz4BHAoxmzQA6Mkp4UlKe8qjsgl3v5a/obzNig==",
```

This shouldn't have changed without a change to `package.json`.
```suggestion
			"version": "1.2.1",
			"resolved": "https://registry.npmjs.org/@preact/signals-core/-/signals-core-1.3.1.tgz",
			"integrity": "sha512-aqzRMNFU1hoQyP4Kb1ldJrUTCnA9vqPDa7qHEQzHJ3upnBOcC2pjmvjAuTqGuY4AVTtUkCQV0FvOCuIQQ2hSdA==",
```